dwave_networkx/drawing/chimera_layout.py

```diff
@@ -28,33 +28,34 @@
 
 
 def chimera_layout(G, scale=1., center=None, dim=2):
-    """Positions the nodes of graph G in a Chimera cross topology.
+    """Position the nodes of graph ``G`` in a Chimera cross topology.
 
     NumPy (https://scipy.org) is required for this function.
 
     Parameters
     ----------
     G : NetworkX graph
-        Should be a Chimera graph or a subgraph of a
-        Chimera graph. If every node in G has a `chimera_index`
-        attribute, those are used to place the nodes. Otherwise makes
-        a best-effort attempt to find positions.
+        :term:`Chimera` :term:`graph` or :term:`subgraph` of a
+        Chimera graph. If every node in ``G`` has a ``chimera_index``
+        attribute, the node position in the ``chimera_index``
+        attribute is used to place each node. Otherwise,
+        a best-effort attempt is made to find the node positions.
 
     scale : float (default 1.)
-        Scale factor. When scale = 1,  all positions fit within [0, 1]
+        Scale factor. If ``scale = 1``, then all positions fit within [0, 1]
         on the x-axis and [-1, 0] on the y-axis.
 
     center : None or array (default None)
         Coordinates of the top left corner.
 
     dim : int (default 2)
-        Number of dimensions. When dim > 2, all extra dimensions are
+        Number of dimensions. If ``dim > 2``, then all extra dimensions are
```

Comment:
Isn't the mathematical symbols more succinct and audience-appropriate?

Reply:
Similar to [this comment](https://github.com/dwavesystems/dwave-networkx/pull/226#discussion_r1098841257), weigh additional formatting versus succinctness of math instead of text. I find it most helpful to look at the rendered HTML for the section to feel if it's starting to look checkerboardish.

Reply:
I think it looks okay.

Reply:
It's also a bit hard in API docs not to have a lot of syntax highlighting; it's just the nature of API docs.